specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json

```diff
@@ -0,0 +1,4472 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "DeviceUpdateClient",
+    "description": "Device Update for IoT Hub is an Azure service that enables customers to publish update for their IoT devices to the cloud, and then deploy that update to their devices (approve updates to groups of devices managed and provisioned in IoT Hub). It leverages the proven security and reliability of the Windows Update platform, optimized for IoT devices. It works globally and knows when and how to update devices, enabling customers to focus on their business goals and let Device Update for IoT Hub handle the updates.",
```

Comment:
I made small update. I am keeping the reference to "Windows Update platform". It is text we also use on the service home documentation/introduction page [https://docs.microsoft.com/en-us/azure/iot-hub-device-update/understand-device-update](https://docs.microsoft.com/en-us/azure/iot-hub-device-update/understand-device-update).

The term `Update` is very common. In fact, in our first preview, we used term `Content` but customers requested us to change to `Update` because that's what they expect.